src/main/java/com/jagrosh/giveawaybot/commands/CreateCommand.java

```diff
@@ -230,18 +230,49 @@ private void waitForPrize(CommandEvent event, TextChannel tchan, int seconds, in
             GuildSettingsManager.GuildSettings settings = bot.getDatabase().settings.getSettings(event.getGuild().getIdLong());
             if (!level.canSetEmoji() && !settings.getEmojiDisplay().equals(Constants.TADA))
             {
-                bot.getDatabase().settings.updateEmoji(event.getGuild(), null);
-                event.reply("Your custom emoji has been reset, because your premium status expired.");
+                event.reply("That is weird. Your custom emoji could not be used.\n\nWould you still like to continue using the `" + Constants.TADA + "` emoji?");
```

Comment:
This sentence feels somewhat strange to read, at least to me. What about "Hmm... Your custom emoji couldn't be used" or "That's weird, Your custom emoji couldn't be used."

Reply:
```code
suggestion
try this and i just click commit
```